Some styling improvements in the documentation (#81)
DSD-DBS/capella-dockerimages: README.md

### Pure::variants

To run the `pure-variants` images, please follow the instructions to run [`Capella in a remote container`](#capella-in-a-remote-container) or [`T4C client in a remote container`](#t4c-client-in-a-remote-container).

You just have to do some changes:

> - Add the environment variable `$PURE_VARIANTS_LICENSE_SERVER` to the `docker run` command. The value is the same that someone has to set in the Capella GUI when running a normal installation, e.g., `http://localhost:8080`.

Reviewer: ```suggestion
- Add the environment variable `$PURE_VARIANTS_LICENSE_SERVER` to the `docker run` command. The value is the same as set in the Capella GUI when running a normal installation, e.g. `http://localhost:8080`.
```
